deployments/scripts/smoke-summoner.sh

```diff
@@ -8,6 +8,15 @@ set -euo pipefail
 # preview is redeployed during the run of this script, the script will fail
 # as the chain ID will be different.
 
+# If `SUMMONER_SMOKE_RESET=1` is set, automatically reset the testnet data
+# directory, and clear the summonerd directory. This is helpful if running this
+# test in a loop with e.g. `entr` or `cargo watch`.
+if [[ "${SUMMONER_SMOKE_RESET:-0}" -eq '1' ]] ; then
+    cargo run --release --bin pd -- testnet unsafe-reset-all
+    rm -rf /tmp/summonerd
+    rm -rf /tmp/account1
+fi
```

Comment:
not strictly tied to the fix, but i found it to be very helpful to have a toggle to clear this state automatically, for the sake of a friendly|tight feedback loop during local development.

Reply:
Big ups. In general, I'd find it more ergonomic for the test suites to use tmpdirs across the board, but iirc pd actually requires a non-existent dir right now. Something to smooth over when next we're editing the pd cli tooling.